tests/beaker_tests/cisco_bridge_domain/test_bridge_domain.rb

```diff
@@ -0,0 +1,290 @@
+###############################################################################
+# Copyright (c) 2015 Cisco and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+###############################################################################
```

Comment:
These tests can be greatly simplified if you use the current test script format. 
For a very simple example, see https://github.com/cisco/cisco-network-puppet-module/blob/develop/tests/beaker_tests/cisco_acl/test_acl.rb
   (note that you don't need to do the title_pattern tests since you only have simple title patterns).